### PR TITLE
US6968 Global Dark Theme

### PR DESCRIPTION
**Pull request by tcmacdonald**
This PR implements a global dark-theme toggle for the DDK. NOTE– the majority of styles to augment elements in the dark-theme are included here (vs. crds–styles) so I'm looking for some input on whether thats appropriate or what.

### CODE REVIEW

File: src/app/directives/theme-toggle-switch/theme-toggle-switch.component.ts [hunk 1]
```diff
@@ -2,14 +2,13 @@ import { Component, EventEmitter, Input, Output  } from '@angular/core';
 
 @Component({
   selector: 'theme-toggle-switch',
-  templateUrl: './theme-toggle-switch.component.html',
-  styleUrls: ['./theme-toggle-switch.component.scss']
+  template: '<a class="theme-toggle-switch" (click)="onSwitch()"><svg class="icon icon-1" viewBox="0 0 256 256"><use height="256" width="256" href="/assets/svgs/icons.svg#contrast"></use></svg></a>'
```
Comment (seancdavis) [inline src/app/directives/theme-toggle-switch/theme-toggle-switch.component.ts]: @tcmacdonald: Why switch to an inline template here?

Comment (tcmacdonald) [inline src/app/directives/theme-toggle-switch/theme-toggle-switch.component.ts]: Seemed weird to have a standalone HTML file w/ just one element but I could be persuaded to revert this, if you think we should.

Comment (seancdavis) [inline src/app/directives/theme-toggle-switch/theme-toggle-switch.component.ts]: I agree with the logic, just curious. 👍

**Comment by tcmacdonald** [comment]
Thanks @seancdavis – after reviewing crdschurch/crds-styles#73 and talking with @zarahzachz I think I'll refactor this a little bit to lean more heavily on the inverted navbar style provided by Bootstrap. I am not convinced these customizations shouldn't be included in `crds-styles` – we just don't have an example of an inverted navbar in use anywhere yet.

**Pull request merged**